src/configuration/rendererSettings.js

```diff
@@ -1,6 +1,27 @@
 export default function rendererSettings() {
     return {
-        id_cols: ['sitename', 'subjectnameoridentifier'],
+        nestings: [
+            {
+                value_col: 'sitename',
+                label: 'Site',
+                default: true
+            },
+            {
+                value_col: 'subjectnameoridentifier',
+                label: 'Subject ID',
+                default: true
+            },
+            {
+                value_col: 'foldername',
+                label: 'Folder',
+                default: false
+            },
+            {
+                value_col: 'formoid',
+                label: 'Form',
+                default: false
```

Comment:
what if all four default to `true`?

Reply:
a sobering question haha, ill probably add what you commented below